src/components/layout/Surface/Surface.tsx

```diff
@@ -86,7 +87,7 @@ const Surface = forwardRef<HTMLDivElement, SurfaceProps>(
     };
 
     return (
-      <SurfaceRoot ref={ref} style={cssVars} {...rest}>
```

Comment:
@julesbarr This was fixed in #873 so you should rebase and drop your change here